### PR TITLE
[ZEPPELIN-1056] Embed webfont when building

### PR DESCRIPTION
**Pull request by corneadoug**
### What is this PR for?
This is some changes to include the previously webfonts when building Zeppelin.
If there is too much problem building (In the case of China), we could eventually make the fonts part of the source code

### What type of PR is it?
Improvement

### Todos
* [x] - Add the fonts licenses (Apache 2.0 + SIL Open Font License v1.10)

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1056

### How should this be tested?
Deactivate the fonts on your computer, build and launch Zeppelin.
The Zeppelin logo should have the Patua One font

### Questions:
* Does the licenses files need update? Yes
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by corneadoug** [comment]
Licenses were added, this is ready for review

**Comment by Leemoonsoo** [comment]
I think `licenses/LICENSE-patuaOne-font` and `licenses/LICENSE-source_code_pro-font` can be moved to `zeppelin-distribution/src/bin_license/licenses`, while font files are not going to be packaged into source release, but only binary release.

**Comment by corneadoug** [comment]
@Leemoonsoo Just moved them

File: .gitignore [hunk 1]
```diff
@@ -36,6 +36,9 @@ reports
 zeppelin-web/node_modules
 zeppelin-web/dist
 zeppelin-web/.tmp
+zeppelin-web/src/fonts/Roboto*
+zeppelin-web/src/fonts/Source-Code-Pro*
+zeppelin-web/src/fonts/Patua-One*
```
Comment (Leemoonsoo) [inline .gitignore]: now ci fails with 

```
[ERROR] Failed to execute goal org.apache.rat:apache-rat-plugin:0.11:check (verify.rat) on project zeppelin-web: Too many files with unapproved license: 6 See RAT report in: /home/travis/build/apache/zeppelin/zeppelin-web/target/rat.txt -> [Help 1]
```

and zeppelin-web/target/rat.txt is

```
6 Unknown Licenses

*******************************

Unapproved licenses:

  src/fonts/Patua-One.css
  src/fonts/Patua-One.svg
  src/fonts/Roboto.css
  src/fonts/Roboto.svg
  src/fonts/Source-Code-Pro.css
  src/fonts/Source-Code-Pro.svg
```

And i think above 3 lines can be moved from `.gitignore` to [apache-rat-plugin exclude section](https://github.com/apache/zeppelin/blob/master/zeppelin-web/pom.xml#L78)

**Comment by Leemoonsoo** [comment]
Great. LGTM!

**Pull request closed without merging**